README.md

```diff
@@ -414,6 +414,8 @@ nativeAlert: ({
 <kbd>App version >=14.10 `withDismiss`, `duration` and `action` in
 response</kbd>.
 
+<kbd>App version >=24.6 `buttonAccessibilityLabel` support</kbd>.
```

Comment:
min app version confirmation is still pending (https://github.com/Telefonica/webview-bridge/issues/132#issuecomment-2159914520)